homeassistant/components/media_player/sonos.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, hass, player):
         self.hass = hass
         self.volume_increment = 5
         self._player = player
+        self._name = None
```

Comment:
@MartinHjelmare ok, now I understand it. Thank you for clarification! I rest my case :)